helsingborg-stad/helsingborg-viva-api-adapter
# Feat/completions description

## Explain the changes you’ve made
I've added support for displaying completions description

## How to test the changes?
1. Checkout branch

2. Run:
```bash
$ cd project-dir
$ pipenv install --dev
$ ./server.sh
```

3. Hit endpoint in Postman or eq
```bash
http://localhost:5000/mypages/AVw1Dxv08eaogAb1lLEOy79EqkK3LOj2/workflows/EABEC231160952E9C125880D005416F1/completions
```

4. Verify description in response
```json
{
  "type": "getWorkflowCompletions",
  "attributes": {
    "requested": [
      {
        "description": "Kvitto på betald hyra",
        "received": true
      },
      {
        "description": "Kvitto på betald elräkning",
        "received": true
      },
      {
        "description": "Underlag på alla sökta utgifter",
        "received": true
      },
      {
        "description": "Underlag på alla inkomster/tillgångar",
        "received": false
      },
      {
        "description": "Alla kontoutdrag för hela förra månaden och fram till idag",
        "received": false
      }
    ],
    "description": "Du behöver skicka in mer",
    "receivedDate": 1652220000000,
    "dueDate": 1652220000000,
    "attachmentUploaded": [],
    "isCompleted": false,
    "isRandomCheck": false,
    "isAttachmentPending": false,
    "isDueDateExpired": true
  }
}
```

Bonus
Run tests
```bash
$ pipenv shell
$ pytest -vv
```
